virshPool.py

```diff
@@ -32,7 +34,13 @@ def ensure_removed(self) -> None:
 
     def initialize(self) -> None:
         print(f"\tInitializing pool {self._name} at {self._images_path}")
-        print(self._host.run(f"virsh pool-define-as {self._name} dir - - - - {self._images_path}"))
-        print(self._host.run(f"mkdir -p {self._images_path}"))
-        print(self._host.run(f"chmod a+rw {self._images_path}"))
-        print(self._host.run(f"virsh pool-start {self._name}"))
+        ret = self._host.run(f"virsh pool-define-as {self._name} dir - - - - {self._images_path}")
```

Comment:
create a function (inline) that wraps self._host.run(...) and exists if returncode != 0

something along the lines of:
```
def run_or_die(cmd):
  ret = self._host.run(cmd)
  if ret.returncode != 0:
     print(f"\tUnable to initialize pool {self._name}: {ret.err}")
     sys.exit(-1)
run_or_die(f"mkdir -p {self._images_path}")
run_or_die(f"chmod a+rw {self._images_path}")
...